services/ux-backend/main.go

```diff
@@ -61,8 +62,15 @@ func main() {
 		klog.Info("shutting down!")
 		os.Exit(-1)
 	}
+	//TODO: deprecate in future release
 	http.HandleFunc("/onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
-		onboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
+		clientonboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
+	})
+	http.HandleFunc("/client-onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
+		clientonboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
+	})
+	http.HandleFunc("/peer-onboarding-tokens", func(w http.ResponseWriter, r *http.Request) {
+		peeronboardingtokens.HandleMessage(w, r, config.tokenLifetimeInHours)
```

Comment:
I am not sure it makes sense that the 2 different onboarding tokens are different REST resources. 
In my mind, it makes more sense that these are 2 sub-resource of a single resource type. 
So in that case the endpoint should be modeled as:

`/onboarding-tokens/client-tokens/` and `/onboarding-tokens/peer-tokens/`
In this case, the mapping of the endpoint `/onboarding-tokens` is to `onboarding-tokens/client-tokens`

This also means that you need to change the folder accordingly